src/openrct2/management/Award.cpp

```diff
@@ -598,6 +603,17 @@ void AwardReset()
     getGameState().currentAwards.clear();
 }
 
+void AwardAdd(AwardType type);
```

Comment:
Duplicate?

Reply:
Btw `missing-declarations` doesn't raise this with the default osx toolchain. Might want to add ` -Wmissing-prototypes` here https://github.com/OpenRCT2/OpenRCT2/blob/develop/CMakeLists.txt#L358 based on https://reviews.llvm.org/D119361.